tests/js/client/restart/test-restart-replication2-view-recovery-cluster.js

```diff
@@ -49,7 +49,7 @@ const waitForServersToBeInCurrent = () => {
     }
   });
   // Sleep a bit, to let AgencyCache see the update
-  require("internal").wait(0.2);
```

Comment:
Note To reviewer:
This fix is unrelated to the PR, this is flakyness of the test.

We unfortunatly cannot wait from the outside if the Coordinator has updated to latest agency cache.